react/features/base/tracks/reducer.js

```diff
@@ -148,3 +149,21 @@ ReducerRegistry.register('features/base/no-src-data', (state = {}, action) => {
     }
 });
 
+/**
+ * Reduces the Redux actions of the feature features/screen-share.
+ */
+ReducerRegistry.register('features/screen-share', (state = {}, action) => {
```

Comment:
I think we don't need this flag at all. We can infer it by checking if we have a track of the desktop kind.